### PR TITLE
implemented vendor update

### PR DESCRIPTION
**Pull request by droot**

### CODE REVIEW

File: cmd/kubebuilder/v1/commands.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ import (
 
 func AddCmds(cmd *cobra.Command) {
 	AddAPICommand(cmd)
+	cmd.AddCommand(vendorUpdateCmd())
```
Comment (Liujingfang1) [inline cmd/kubebuilder/v1/commands.go]: You can remove the `TODO` item.

Comment (droot) [reply cmd/kubebuilder/v1/commands.go]: Done.

**Comment by droot** [comment]
@mengqiy Yes, test needs to be added. Will do that in a separate PR.

**Pull request merged**